src/tests/ftest/ior/ior_small.yaml

```diff
@@ -31,6 +31,8 @@ pool:
     scm_size: 3000000000
     nvme_size: 9000000000
     control_method: dmg
+dfuse:
+    disable_caching: true
```

Comment:
We should probably remove this since a newer commit adds `disable_caching` lower in this config

Reply:
I thought I had, just about to re-push this so I'll do it next time.

Reply:
Oh, you're right, you did. GitHub was lying to me :(